modules/standard/ChapelIO.chpl

```diff
@@ -885,6 +891,7 @@ module ChapelIO {
     this.init(idxType, bounds, strides);
     this.readThis(reader);
   }
+  range implements initDeserializable;
```

Comment:
```suggestion
  implements initDeserializable(range)
```
might be better for this and other (generic with default) types, so that the interface is declared as generically as possible.